macosx/AboutWindowController.h

```diff
@@ -2,7 +2,7 @@
 // It may be used under the MIT (SPDX: MIT) license.
 // License text can be found in the licenses/ folder.
 
-#import <Cocoa/Cocoa.h>
+#import <AppKit/AppKit.h>
```

Comment:
Seems to be unrelated to the title of the PR?

Reply:
Correct. Reverting.